src/FindAnApprenticeship/SFA.DAS.FindAnApprenticeship/InnerApi/Requests/GetApprenticeshipCountRequest.cs

```diff
@@ -1,10 +1,11 @@
+using SFA.DAS.FindAnApprenticeship.Domain.Models;
 using SFA.DAS.SharedOuterApi.Interfaces;
 
 namespace SFA.DAS.FindAnApprenticeship.InnerApi.Requests
 {
-    public class GetApprenticeshipCountRequest : IGetApiRequest
+    public class GetApprenticeshipCountRequest(WageType? wageType = null) : IGetApiRequest
     {
-        public string GetUrl => "/api/vacancies/count";
+        public string GetUrl => $"/api/vacancies/count?wageType={wageType}";
```

Comment:
Careful adding null values into query string parameters, they can sometimes get interpreted in unpredictable ways depending on the type.

Reply:
Used StringBuilder to construct the Url, but we have to come up with a QueryBuilderExtension class to handle it through out the application since we heavy use this kind of Url builder all over the place.